src/FixerDefinition/CodeSampleInterface.php

```diff
@@ -26,4 +26,9 @@ public function getCode();
      * @return null|array
      */
     public function getConfiguration();
+
+    /**
+     * @return null|int
+     */
+    public function getRequiredVersion();
```

Comment:
Maybe we should add an upper and lower bound, as some fixers fix code not valid on PHP7 for example.